### ISSUE TITLE
Set event loop explicity for test_utils.AioHTTPTestCase

### ISSUE THREAD

**Issue by dennypenta** [aio-libs/aiohttp#2268]
## Long story short

I should mock service that depends on event loop or set event loop explicity in statement like so:
```
@classmethod
    def setUpClass(cls):
        cls.loop = asyncio.get_event_loop()
```
## Expected behaviour

AioHTTPTestCase do not close event loop while psycopg connection is underway

## Actual behaviour

`psycopg2.ProgrammingError: close cannot be used while an asynchronous query is underway`

## Steps to reproduce

run more then 1 aiopg coroutine in 1 unittest
for example:
```
        @unittest_run_loop
        async def test_create_instance(self):
            instance = await Model.create_instance({})

        @unittest_run_loop
        async def test_retrieve_instance(self):
            instance = await Model.create_instance({})
```

## Your environment

Mac OS Sierra 10.12.5
aiohttp (2.2.5)
aiopg (0.13.0)
psycopg2 (2.7.3.1)

What I do wrong?
Thanks for your attention.

**Comment by asvetlov** [comment]
Sorry, I don't follow. 
What is your problem?

**Comment by dennypenta** [comment]
I'm so apologizing to you.

The problem was solved in the following way:
    my problem was that my tests of modules with peewee_async was falling with the error `peewee.ProgrammingError: close can not be used while an asynchronous query is underway`, and the call stack is written in exceptions from aiopg and psycopg2.

I first encountered this problem using pytest, uvloop.

Further looking through source code I was confused by `asyncio.set_event_loop (None)`
In general, the problem was not in AioHTTPTestCase and not in pytest-aiohttp.

The fact is that using an asynchronous wrapper to the database made explicit control over opening / closing connections (pool management), but made these calls synchronously, that is, closing.
Reason for this was that Event loop was closed early then database connection.

Once again I apologize.
